<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNewGWASProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionVisualize"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
    <addaction name="actionRemoveAll"/>
   </widget>
   <widget class="QMenu" name="menuAnalyse">
    <property name="title">
     <string>Analyse</string>
    </property>
    <addaction name="actionGLM"/>
    <addaction name="actionEMMAX"/>
    <addaction name="actionGEMMA"/>
    <addaction name="separator"/>
    <addaction name="actionPCA"/>
    <addaction name="separator"/>
    <addaction name="actionStructure"/>
    <addaction name="actionFastStructure"/>
    <addaction name="separator"/>
    <addaction name="actionKinship"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVisualization">
    <property name="title">
     <string>Visualization</string>
    </property>
    <addaction name="actionGWASVisualization"/>
    <addaction name="actionGWASQQVisualization"/>
    <addaction name="actionPCAVisualization"/>
    <addaction name="actionStructureVisualization"/>
    <addaction name="actionKinshipVisualization"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalyse"/>
   <addaction name="menuVisualization"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewGWASProject"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionRemove"/>
   <addaction name="actionRemoveAll"/>
   <addaction name="separator"/>
   <addaction name="actionVisualize"/>
   <addaction name="separator"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionNewGWASProject">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/newProjectIcon.png</normaloff>
     <disabledoff>:/icon/newProjectIcon.png</disabledoff>:/icon/newProjectIcon.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>new project</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/removeProjectIcon.png</normaloff>:/icon/removeProjectIcon.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>remove</string>
   </property>
  </action>
  <action name="actionEMMAX">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/GWASIcon.png</normaloff>:/icon/GWASIcon.png</iconset>
   </property>
   <property name="text">
    <string>EMMAX</string>
   </property>
  </action>
  <action name="actionGEMMA">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/GWASIcon.png</normaloff>:/icon/GWASIcon.png</iconset>
   </property>
   <property name="text">
    <string>GEMMA</string>
   </property>
  </action>
  <action name="actionGLM">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/GWASIcon.png</normaloff>:/icon/GWASIcon.png</iconset>
   </property>
   <property name="text">
    <string>GLM</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actiontest2">
   <property name="text">
    <string>test2</string>
   </property>
  </action>
  <action name="actionFastPCA">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/PCAIcon.png</normaloff>:/icon/PCAIcon.png</iconset>
   </property>
   <property name="text">
    <string>FastPCA</string>
   </property>
  </action>
  <action name="actionPCA">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/PCAIcon.png</normaloff>:/icon/PCAIcon.png</iconset>
   </property>
   <property name="text">
    <string>PCA</string>
   </property>
  </action>
  <action name="actionStructure">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/structureIcon.png</normaloff>:/icon/structureIcon.png</iconset>
   </property>
   <property name="text">
    <string>STRUCTURE</string>
   </property>
  </action>
  <action name="actiontest3">
   <property name="text">
    <string>test3</string>
   </property>
  </action>
  <action name="actionKinship">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/KinshipIcon.png</normaloff>:/icon/KinshipIcon.png</iconset>
   </property>
   <property name="text">
    <string>Kinship</string>
   </property>
  </action>
  <action name="actionGWASVisualization">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/GWASIcon.png</normaloff>:/icon/GWASIcon.png</iconset>
   </property>
   <property name="text">
    <string>GWAS Visualization</string>
   </property>
  </action>
  <action name="actionPCAVisualization">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/PCAIcon.png</normaloff>:/icon/PCAIcon.png</iconset>
   </property>
   <property name="text">
    <string>PCA Visualization</string>
   </property>
   <property name="toolTip">
    <string>PCA Visualization</string>
   </property>
  </action>
  <action name="actionStructureVisualization">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/structureIcon.png</normaloff>:/icon/structureIcon.png</iconset>
   </property>
   <property name="text">
    <string>Structure Visualization</string>
   </property>
  </action>
  <action name="actionKinshipVisualization">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/KinshipIcon.png</normaloff>:/icon/KinshipIcon.png</iconset>
   </property>
   <property name="text">
    <string>Kinship Visualization</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/removeAll.png</normaloff>:/icon/removeAll.png</iconset>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="toolTip">
    <string>remove all project</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/openFileIcon.png</normaloff>:/icon/openFileIcon.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>load file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/saveIcon.png</normaloff>:/icon/saveIcon.png</iconset>
   </property>
   <property name="text">
    <string>Save ...</string>
   </property>
   <property name="toolTip">
    <string>save result</string>
   </property>
  </action>
  <action name="actionVisualize">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/visualizeIcon.png</normaloff>:/icon/visualizeIcon.png</iconset>
   </property>
   <property name="text">
    <string>Visualization</string>
   </property>
   <property name="toolTip">
    <string>result visualization</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/pauseIcon.png</normaloff>:/icon/pauseIcon.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/stopIcon.png</normaloff>:/icon/stopIcon.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>stop running</string>
   </property>
  </action>
  <action name="actionFastStructure">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/structureIcon.png</normaloff>:/icon/structureIcon.png</iconset>
   </property>
   <property name="text">
    <string>FastStructure</string>
   </property>
  </action>
  <action name="actionNormalBSA">
   <property name="text">
    <string>BSA</string>
   </property>
  </action>
  <action name="actionGWASQQVisualization">
   <property name="icon">
    <iconset resource="../Res.qrc">
     <normaloff>:/icon/QQPlotIcon.png</normaloff>:/icon/QQPlotIcon.png</iconset>
   </property>
   <property name="text">
    <string>Q-Q Visualization</string>
   </property>
   <property name="toolTip">
    <string>Start GWAS QQ Visualization</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections/>
</ui>
